docs/snippets/svelte/component-story-static-asset-with-import.native-format.mdx

```diff
@@ -0,0 +1,27 @@
+```html
+<!-- MyComponent.stories.svelte -->
+
+<script>
+  import { Meta, Template, Story } from '@storybook/addon-svelte-csf';
+
+  import MyComponent from './MyComponent.svelte';
+
+  import imageFile from './static/image.png';
+
+  let image = {
+    src: imageFile,
+    alt: 'my image',
+  };
+</script>
+
+<Meta
+  title="img"
+  component={MyComponent}
+/>
+
+<Template >
+  <MyComponent image={image} />
```

Comment:
```suggestion
  <MyComponent {image} />
```

`MyComponent` has here an image property, and sometimes in others examples a src and alt properties. it's look strange 😄

Reply:
Fixed with the latest commit